modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -2121,6 +2141,9 @@ void GDScriptAnalyzer::reduce_call(GDScriptParser::CallNode *p_call, bool is_awa
 
 		if (is_self && parser->current_function != nullptr && parser->current_function->is_static && !is_static) {
 			push_error(vformat(R"*(Cannot call non-static function "%s()" from static function "%s()".)*", p_call->function_name, parser->current_function->identifier->name), p_call->callee);
+		} else if (!is_self && base_type.is_meta_type && !is_static) {
+			base_type.is_meta_type = false; // For `to_string()`.
+			push_error(vformat(R"*(Cannot call non-static function "%s()" on the class "%s" directly. Make an instance instead.".)*", p_call->function_name, base_type.to_string()), p_call->callee);
```

Comment:
I'm still getting confused by the raw string syntax, but isn't there an extraneous `."` at the end of the string here?

Reply:
Ah, indeed. I probably added it mechanically.